sdcm/utils/version_utils.py

```diff
@@ -117,6 +118,12 @@ class ScyllaFileType(Enum):
 RepositoryDetails = namedtuple("RepositoryDetails", ["type", "urls"])
 
 
+@property
+def parse_scylla_version(version_to_parse: str) -> version.Version:
```

Comment:
Maybe it should handle also `latest` tag/version?

Reply:
it is called only when we have already a version selected... it doesn't matter to us what we are passing to the pipeline, prior to having an image selected, but this is for version comparisons only, when we need to know if a feature we want to test is already in that version we are running on

Reply:
and BTW, now there is a change that will get either `digit.digit` or `digit.digit.digit`, so `5.2.latest` or `5.3.dev` will be covered by it now